anagram.cpp

```diff
@@ -1,13 +1,50 @@
 #include "anagram.h"
+#include <string>
+#include <vector>
+#include <bits/stdc++.h>
+#include <algorithm>
+
+using namespace std;
 
 bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2) {
-    //Fill the correct implementation here
+    
+    string word1Copy = word1;
+    string word2Copy = word2;
+    
+    //word1Copy.erase(remove(word1Copy.begin(), word1Copy.end(), ' '), word1Copy.end());
+    //word2Copy.erase(remove(word2Copy.begin(), word2Copy.end(), ' '), word2Copy.end());
+    
```

Comment:
Impressive removal of non-alphabets 👍 
Don't leave commented code in commits. If we need the old code, we can always look it up in the history.

Reply:
Yes, I removed the comments.